server.py

```diff
@@ -327,8 +327,11 @@ def server_points(server):
 					else:
 						points += 1
 			else:
-				# Old server (1/4 per client)
-				points = server["clients"] / 4
+				for name in server["clients"]:
```

Comment:
Sorry? `clients` is a numeric value since.. ever.
https://github.com/minetest/minetest/blame/b2dfde8c8c29d0b118ba9018be202de21bb28505/src/serverlist.cpp

Reply:
You right. I have not studied this part of the code and made an erroneous proposal.